frontend/src/index.css

```diff
@@ -93,6 +93,12 @@ Other button classes are defined further down together with other classes for th
   align-items: center;
 }
 
+@media (max-width: 700px) and (orientation: portrait) {
+  .header-wrapper {
+    width: 80vh;
```

Comment:
Do you know that there is another unit `vw` that refers to the width? (short for viewport width, whereas `vh` is viewport height). 
I think it would make more sense to use that one here and everywhere further down where you define the width. Otherwise, the _width_ of the page is depending on the _height_ of the device, which doesn't really make sense, because devices can have different proportions. 

(We might have done the same thing before, but if that's the case this could be a good point to fix even earlier mistakes :) )